checkov/terraform/graph_builder/foreach_handler.py

```diff
@@ -143,5 +149,75 @@ def _build_sub_graph(self, blocks_to_render: list[int]) -> l_graph.TerraformLoca
         sub_graph.out_edges = self.local_graph.out_edges
         return sub_graph
 
+    def _create_new_resources_count(self, statement: int, main_resource: TerraformBlock) -> None:
+        for i in range(statement):
```

Comment:
`statement` does not sound like a name of an integer, is this naming part of tf foreach/count naming I'm not familiar with? If not, please rename it

Reply:
The func name is for `count`, a count statement is a number -

> resource "aws_s3_bucket" "count_var_resource" {
  `count = 5`
  name     = count.index
}